locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -2297,7 +2297,6 @@ protected JcloudsWinRmMachineLocation createWinRmMachineLocation(ComputeService
                     .configure("jcloudsParent", this)
                     .configure("displayName", vmHostname)
                     .configure("address", address)
-                    .configure(WinRmMachineLocation.WINRM_PORT, sshHostAndPort.isPresent() ? sshHostAndPort.get().getPort() : node.getLoginPort())
```

Comment:
This is creating the `WinRmMachineLocation` and its `getPort` will look for `WINRM_CONFIG_PORT` key, but you are not passing it any.